src/test/cbmc/LoopWhile_NonReturning/main.rs

```diff
@@ -1,5 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// cbmc-flags: --unwind 11 --unwinding-assertions
```

Comment:
This was not failing before because we had not included `--unwinding-assertions`, right?

Reply:
Yes.